### ISSUE TITLE
UnhandledPromiseRejectionWarning

### ISSUE THREAD

**Issue by Nebu2k** [mdaskalov/homebridge-dreambox#10]
I get the following error when I try to switch channel or do any other interaction.

(node:1511) UnhandledPromiseRejectionWarning: Error: This callback function has already been called by someone else; it can only be called one time.
    at /usr/local/lib/node_modules/homebridge/node_modules/hap-nodejs/src/lib/util/once.ts:6:13
    at /usr/local/lib/node_modules/homebridge-dreambox/dreambox-accessory.js:209:21
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:94:5)
(node:1511) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:1511) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
(node:1511) UnhandledPromiseRejectionWarning: Error: This callback function has already been called by someone else; it can only be called one time.
    at /usr/local/lib/node_modules/homebridge/node_modules/hap-nodejs/src/lib/util/once.ts:6:13
    at /usr/local/lib/node_modules/homebridge-dreambox/dreambox-accessory.js:209:21
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:94:5)
(node:1511) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 2)

Any idea?

**Comment by raybanz** [comment]
I have the same issue when power on or off.
This callback function has already been called by someone else; it can only be called one time.
    at /usr/local/lib/node_modules/homebridge/node_modules/hap-nodejs/src/lib/util/once.ts:6:13
    at /usr/local/lib/node_modules/homebridge-dreambox/dreambox-accessory.js:209:21
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:709) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:709) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
(node:709) UnhandledPromiseRejectionWarning: Error: This callback function has already been called by someone else; it can only be called one time.
    at /usr/local/lib/node_modules/homebridge/node_modules/hap-nodejs/src/lib/util/once.ts:6:13
    at /usr/local/lib/node_modules/homebridge-dreambox/dreambox-accessory.js:209:21
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:709) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 2)

**Comment by mdaskalov** [comment]
Could you please specify what image is installed on your dreambox? 
Is reading of the channels from the configured bouquet working?

To switch **on** call:
`http://{dream_ip}/web/powerstate?newstate=4`
To switch **off** call:
`http://{dream_ip}/web/powerstate?newstate=5`

The answer should be:
```
<?xml version="1.0" encoding="UTF-8"?>
<e2powerstate>
	<e2instandby> {true | false} </e2instandby>
</e2powerstate>
```
Please write down the answer you get (if different), so I can reproduce the error.

**Comment by Nebu2k** [comment]
It's a NewNigma2 Image.
Reading of the channels works fine!

http://{dream_ip}/web/powerstate?newstate=4
http://{dream_ip}/web/powerstate?newstate=5
also works fine via browser.

**Comment by raybanz** [comment]
OpenPli 7.2
Bouquets reading is working on all channel.

switch on call:

<?xml version="1.0" encoding="UTF-8"?>
<e2powerstate>
	<e2instandby>
false	</e2instandby>
</e2powerstate>


switch off call:

<?xml version="1.0" encoding="UTF-8"?>
<e2powerstate>
	<e2instandby>
true	</e2instandby>
</e2powerstate>

**Comment by raybanz** [comment]
https://github.com/E2OpenPlugins/e2openplugin-OpenWebif/wiki/OpenWebif-API-documentation#powerstate <https://github.com/E2OpenPlugins/e2openplugin-OpenWebif/wiki/OpenWebif-API-documentation#powerstate>

**Comment by mdaskalov** [comment]
There was a problem in the URL encoding - the '?' char got URL-escaped.
Should be fixed it in v0.4.5 please update.
